crates/forge_analyzer/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+use crate::definitions::CalleeRef;
+use swc_core::ecma::ast::{CallExpr, Callee, Expr, MemberProp};
+
+pub fn call_func_with_name(n: &CalleeRef, name: &str) -> bool {
+    if let CalleeRef::Expr(expr) = &n {
```

Comment:
```suggestion
    if let CalleeRef::Expr(Expr::Member(mem)) = n {
```
You can take advantage of [match ergonomics](https://rust-lang.github.io/rfcs/2005-match-ergonomics.html).